app/views/users/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML FORM</title>
+</head>
+
+<body>
```

Comment:
We don't need this HTML code here. We already have this code in the application.html.erb file